packages/__docs__/src/Properties/index.js

```diff
@@ -92,6 +107,44 @@ class Properties extends Component {
       })
   }
 
+  renderTSType = (tsType) => {
+    let { name, elements, type, raw } = tsType
+    let isEnum = true
+    /*
+    possible types:
+      - signature
+      - union
+      - custom
+      - boolean
+      - array
+    */
+    // TODO: currently custom imported types are just showing the name of the can somehow link to these custom types
+    switch (name) {
+      case 'union':
+        elements.forEach((element) => {
+          if (element.name !== 'literal') {
+            isEnum = false
+          }
+        })
+        return isEnum ? 'enum' : 'union'
+      case 'signature':
+        switch (type) {
+          case 'function':
+            return 'func'
```

Comment:
I tried to call them the same way as before, also shorter names help a lot with the width of the table column